themes/default/content/docs/intro/concepts/resources/providers.md

```diff
@@ -377,3 +377,24 @@ class MyStack
 {{% /choosable %}}
 
 {{< /chooser >}}
+
+While default providers are enabled by default, they [can be disabled]({{< relref
```

Comment:
Suggest we put this under a section header like `## Disabling Default Providers`.